cms/static/js/utils/date_utils.js

```diff
@@ -35,9 +35,19 @@ define(["jquery", "date", "jquery.ui", "jquery.timepicker"], function($, date) {
         );
     };
 
```

Comment:
As mentioned above, can we include convertDateStringsToObjects() in this utility library?